system-variables.md

```diff
@@ -2155,16 +2155,12 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 
 ### `tidb_load_based_replica_read_threshold` <span class="version-mark">从 v7.0.0 版本开始引入</span>
 
-> **警告：**
->
-> 当前版本中该变量控制的功能尚未完全生效，请保留默认值。
-
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 默认值：`"0s"`
```

Comment:
We are considering changing this value to a proper default enable value.

Reply:
I've talked with @Oreoxmt, the default value of it can be changed until the code freeze, we can just finish tech review in this stage.